SYCL/ESIMD/spec_const/spec_const_ushort.cpp

```diff
@@ -0,0 +1,21 @@
+//==--------------- spec_const_ushort.cpp  - DPC++ ESIMD on-device test ---===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu
+// XFAIL: level_zero || windows
```

Comment:
why level_zero is XFAIL? please file a bug if it does not work on level_zero

Reply:
Some tests (signed/unsigned char and short types) fail on Level Zero with assert:
> /opt/src/build_igc/VectorCompiler/llvm-spirv-vc/lib/SPIRV/SPIRVReader.cpp:1350: llvm::Value* SPIRV::SPIRVToLLVM::transValueWithoutDecoration(SPIRV::SPIRVValue*, llvm::Function*, llvm::BasicBlock*, bool):
> Assertion `(BT->getBitWidth() == 64 || (ConstValue >> BT->getBitWidth()) == 0) && "Size of externally provided specialization constant value doesn't" "fit into the specialization constant type"' failed.

According to specification all basic types should be supported, also these tests pass on OpenCL. [Ticket](https://jira.devtools.intel.com/browse/VMIT-8680)